shared/models/user.ts

```diff
@@ -0,0 +1,13 @@
+export type Coordinates = { latitude: number; longitude: number };
+
+export type UserType = {
+  id?: string;
+  name: string;
+  email: string;
+  password: string;
+  location: Coordinates;
+  photoUri: string;
```

Comment:
Let's make it optional since we don't want to require users to upload a picture while signing up.